src/codeu/chat/ServerMain.java

```diff
@@ -47,21 +47,22 @@ public static void main(String[] args) {
     LOG.info("============================= START OF LOG =============================");
 
     final Uuid id = Uuid.fromString(args[0]);
+    //final Uuid id = Uuid.fromString("1");
     final byte[] secret = Secret.parse(args[1]);
 
     final int myPort = Integer.parseInt(args[2]);
 
     // This is the directory where it is safe to store data accross runs
     // of the server.
-    final String persistentPath = args[3];
+    //final String persistentPath = args[3];
 
     final RemoteAddress relayAddress = args.length > 4 ?
-                                       RemoteAddress.parse(args[4]) :
-                                       null;
+            RemoteAddress.parse(args[4]) :
+            null;
 
     try (
-        final ConnectionSource serverSource = ServerConnectionSource.forPort(myPort);
-        final ConnectionSource relaySource = relayAddress == null ? null : new ClientConnectionSource(relayAddress.host, relayAddress.port)
+            final ConnectionSource serverSource = ServerConnectionSource.forPort(myPort);
```

Comment:
try not to autoIndent, it will look like you have a change even though it is just whitespaces